### PR TITLE
Add deploy component label

### PR DESCRIPTION
**Pull request by fiunchinho**

### CODE REVIEW

**Review by whites11 — APPROVED** [review]
LGTMIIWFY

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
```
Comment (kopiczko) [inline CHANGELOG.md:10]:
```suggestion
### Added
```

**Review by kopiczko — APPROVED** [review]

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -1,43 +1,8 @@
...
-
-<!--
-Changelog must always be updated.
--->
-
 ### Checklist
 
 - [ ] Update changelog in CHANGELOG.md.
-- [ ] Make sure `values.yaml` and `values.schema.json` are valid.
+- [ ] Make sure to run `make generate` to generate Helm manifests.
```
Comment (nprokopic) [inline .github/pull_request_template.md:8]: Not now, but we should add tests that ensure that `make generate` was executed, something like this in CAPI https://github.com/kubernetes-sigs/cluster-api/blob/v1.0.3/Makefile#L699-L704.

File: config/helm/kustomization.yaml [hunk 3]
```diff
@@ -5,14 +5,14 @@ resources:
...
   - name: k8s.gcr.io/cluster-api/kubeadm-bootstrap-controller
-    newName: "{{ .Values.registry.domain }}/{{ .Values.images.bootstrap.name }}"
-    newTag: "{{ .Values.images.tag }}"
+    newName: "{{.Values.registry.domain}}/{{.Values.images.bootstrap.name}}"
+    newTag: "{{.Values.images.tag}}"
   - name: k8s.gcr.io/cluster-api/kubeadm-control-plane-controller
-    newName: "{{ .Values.registry.domain }}/{{ .Values.images.controlplane.name }}"
-    newTag: "{{ .Values.images.tag }}"
+    newName: "{{.Values.registry.domain}}/{{.Values.images.controlplane.name}}"
+    newTag: "{{.Values.images.tag}}"
```
Comment (fiunchinho) [inline config/helm/kustomization.yaml:15]: Removed spaces to avoid https://github.com/kubernetes-sigs/kustomize/issues/947

**Pull request merged**